packages/prism/test/index.tsx

```diff
@@ -2,15 +2,16 @@
  * @jest-environment jsdom
  */
 
-import React, { ReactElement } from 'react'
+import React from 'react'
```

Comment:
@hasparus Could you quickly advise on why https://github.com/system-ui/theme-ui/actions/runs/3063545168/jobs/4945735744#step:7:159 is coming up? It's totally fine if I `ts-expect-error` this import but I can't get it to work…